src/main/java/com/pingcap/tikv/codec/FloatUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.pingcap.tikv.codec;
```

Comment:
But they have different method. This changes was brought by you. I may readd it during resolving conflict.